tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -210,7 +210,8 @@ def test_should_respond_200(self, username, session):
         # This prevents issue when users upgrade to 2.0+
         # from 1.10.x
         # https://github.com/apache/airflow/issues/14421
-        session.query(TaskInstance).update({TaskInstance.operator: None}, synchronize_session="fetch")
+        session.query(TaskInstance).update(
+            {TaskInstance.operator: None}, synchronize_session="fetch")
```

Comment:
Try to avoid unnecessary and unrelated changes
```suggestion
        session.query(TaskInstance).update({TaskInstance.operator: None}, synchronize_session="fetch")
```

Reply:
Easy fix, will turn off my formatter